src/plugins/shared_ux/public/components/toolbar/solution_toolbar/quick_group_buttons/quick_group.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonGroup,
+  htmlIdGenerator,
+  EuiButtonGroupOptionProps,
+  useEuiTheme,
+  IconType,
+} from '@elastic/eui';
+import { QuickGroupButtonStyles } from './quick_group.styles';
+export interface QuickButton {
+  label: string;
+  title?: string;
+  onClick: () => void;
+  iconType: IconType;
+  getLegend: string;
```

Comment:
I was getting very confused by this prop and noticed that only the first button's `getLegend` prop is ever used and yet it's being required for every button. I'm pretty sure the intent here is to actually require one `legend` for the whole group to be passed through to `EuiButtonGroup`. 

I'm pushing you a commit for this too.

Reply:
https://github.com/elastic/kibana/pull/126297/commits/ae3e04ef62bb1e17994d00dbafdb07ac25794c45